### PR TITLE
use new swipe-layout also in new chat

### PR DESCRIPTION
**Pull request by r10s**
as discussed, this uses the new swipe layout from https://github.com/deltachat/deltachat-ios/pull/2351 also for the few remaining lists

### CODE REVIEW

File: deltachat-ios/Controller/NewChatViewController.swift [hunk 1]
```diff
@@ -349,6 +357,7 @@ extension NewChatViewController {
         )
         alert.addAction(UIAlertAction(title: String.localized("delete"), style: .destructive, handler: { [weak self] _ in
             self?.deleteContact(contactId: contactId, indexPath: indexPath)
+            callback?()
```
Comment (Amzd) [inline deltachat-ios/Controller/NewChatViewController.swift:360]: callback is not called on all paths which means the UIContextualAction's completionHandler won't be called when you cancel.

Comment (r10s) [reply deltachat-ios/Controller/NewChatViewController.swift:360]: is it really needed that the completion handler is always called. 

how does eg. WhatsApp keep a swipe-action open cancelling a subsequent dialog (eg. at "more" there)

Comment (Amzd) [reply deltachat-ios/Controller/NewChatViewController.swift:360]: AH so it is intentional, mb

Comment (r10s) [reply deltachat-ios/Controller/NewChatViewController.swift:360]: still, the name `callback()` is not-so-great, maybe better `confirmed()`

Comment (Amzd) [reply deltachat-ios/Controller/NewChatViewController.swift:360]: Ah yeah or `didDelete()`

**Pull request merged**